bin/start_tests.sh

```diff
@@ -11,8 +16,13 @@ deactivate
 # Move down to the dcoscli package
 cd cli
 
-cp tests/data/dcos.toml $DCOS_CONFIG
-echo "$VBOX_IP dcos.snakeoil.mesosphere.com" | sudo tee -a /etc/hosts > /dev/null
+# overwrite core.dcos_url in the dcos.toml
+cp ./tests/data/dcos.toml $DCOS_CONFIG
+sed -i "s/change.dcos.url/$DCOS_URL/g" $DCOS_CONFIG
```

Comment:
I thought we change the code to just use `dcos.snakeoil.mesosphere.com` so we didn't have to do this, no?